frontend/packages/console-plugin-sdk/src/typings/kebab-actions.ts

```diff
@@ -1,11 +1,11 @@
-import { K8sKind } from '@console/internal/module/k8s';
+import { K8sKind, K8sResourceKind } from '@console/internal/module/k8s';
 import { KebabAction } from '@console/internal/components/utils/kebab';
 import { Extension } from './base';
 
 namespace ExtensionProperties {
   export interface KebabActions {
     // no kind should return any actions common for all kinds
-    getKebabActionsForKind: (kind: K8sKind) => KebabAction[];
+    getKebabActionsForKind: (kind: K8sKind, resource?: K8sResourceKind) => KebabAction[];
```

Comment:
@vojtechszocs we need to add flag as well in order to gate the enabling of fetaure as per the ocs version. I did not add as I wasnt sure about the use case of the existing "KebabActions" extension.

Reply:
Added